src/test/incremental/hashes/trait_defs.rs

```diff
@@ -78,7 +78,7 @@ trait TraitAddReturnType {
 }
 
 #[cfg(not(cfail1))]
-#[rustc_clean(label="hir_owner", cfg="cfail2")]
+#[rustc_dirty(label="hir_owner", cfg="cfail2")]
```

Comment:
What does this change mean?
Also, are all uses of `-Zincremental-ignore-spans` eliminated from tests?